swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -7834,11 +8007,11 @@ components:
       type: object
       additionalProperties: 
         $ref: "#/components/schemas/hrefType"
-      required:
-        - cardAccount
```

Comment:
Relevant Change:  cardAccount is now required
Adoption approach: Follow the change